readthedocs/projects/templatetags/projects_tags.py

```diff
@@ -25,3 +25,9 @@ def sort_version_aware(versions):
 def is_project_user(user, project):
     """Return if user is a member of project.users."""
     return user in project.users.all()
+
+
+@register.filter
```

Comment:
Is there a reason why this is a filter rather than just data set on the context in `AccountDelete`? It seems a somewhat awkward filter.

Reply:
Sorry for the late reply. After reading the code again and the one on .com, I think it was because we need to override this on .com, but I'll see if I can use the SetttingsObjectOverride pattern here

Reply:
Managed to get it working with SettingsOverride :)